docs/sql/scalar.txt

```diff
@@ -144,11 +144,82 @@ If the time zone is not specified, truncation is based on UTC time::
     +---------------+---------------+
     SELECT 3 rows in set (... sec)
 
+extract(field from source) returns long
```

Comment:
extract is no scalar function but an expression like match, maybe list it elsewhere in a separate section?